CoffeeChat/Controller/AboutPearViewController.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  AboutPearViewController.swift
+//  CoffeeChat
+//
+//  Created by Amy Chin Siu Huang on 10/11/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class AboutPearViewController: UIViewController {
+    
+    // MARK: - Private View Vars
+    private let aboutLabel = UILabel()
+    private let aboutTableView = UITableView()
+    private let backButton = UIButton()
+    private let backgroundImage = UIImageView(frame: UIScreen.main.bounds)
+    private let feedbackButton = UIButton()
+    private let moreAppsButton = UIButton()
+    private let settingsTableView = UITableView()
+    private let visitWebsiteButton = UIButton()
+    
+    // MARK: - Private Data Vars
+    private let aboutParagraphs: [AboutParagraph] = [
+        AboutParagraph(bold: "Get paired ", regular: "up with a new Cornell student like you, every week"),
+        AboutParagraph(bold: "Reach out ", regular: "by sending over when and where you prefer, or choose from what your pair suggested."),
+        AboutParagraph(bold: "Meet at ", regular: "the chosen time and place, and now you have one new friend at Cornell!"),
+        AboutParagraph(bold: "", regular: "You can pause pairings at any time in Settings.")
+    ]
+    let aboutReuseId = "aboutReuseId"
```

Comment:
I think we have decided to start using the convention where you set `aboutReuseId` inside of the `AboutPearTableViewCell` class. So you say `static let reuseIdentifier = "AboutReuseIdentifier"` inside of the cell's class and then when you have to use it, you call `AboutPearTableViewCell .reuseIdentifier`.